packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -57,6 +58,14 @@ export function buildPackageFile(
     packageInfo = buildAzureStandalonePackage(extendedConfig);
   }
 
+  if (
+    model.options?.moduleKind == "esm" &&
+    model.options.isModularLibrary &&
```

Comment:
do we have metadata in RLC?

Reply:
yes, if enable azureSdkForJs, metadata will be generated in typespec RLC, 
see https://github.com/Azure/autorest.typescript/pull/2888/files#diff-d58a1b40354ea90a08e1f21d4c6c55fd325197c24d6da40c2d96028a29fe4b16R474-R483